src/Randomizer.SMZ3.Tracking/VoiceCommands/ChatIntegrationModule.cs

```diff
@@ -167,6 +178,75 @@ public async Task DeclareGanonsTowerGuessingGameWinner(int winningNumber)
             }
         }
 
+        /// <summary>
+        /// Asks the chat if tracker should increase content by one step
+        /// </summary>
+        /// <returns></returns>
+        public async Task AskChatAboutContent()
+        {
+            // Always ask the first time, otherwise it's a random change. Can probably change to always be random later.
+            var shouldAskChat = !HasAskedChatAboutContent || s_random.Next(0, 4) == 0;
+            if (!ShouldCreatePolls || !shouldAskChat)
+            {
+                Tracker.TrackItem(Tracker.Items.Where(x => x.Name.First() == "Content").First());
+                return;
+            }
+
+            AskChatAboutContentPollId = await ChatClient.CreatePollAsync("Do you think that was some high quality #content?", new List<string>() { "Yes", "No" }, AskChatAboutContentPollTime);
+
+            if (AskChatAboutContentPollId == null)
+            {
+                Tracker.TrackItem(Tracker.Items.Where(x => x.Name.First() == "Content").First());
+                return;
+            }
+
+            Tracker.Say(x => x.Chat.AskChatAboutContent);
+            Tracker.Say(x => x.Chat.PollOpened, AskChatAboutContentPollTime);
+            AskChatAboutContentCheckPollResults = true;
+            HasAskedChatAboutContent = true;
+            var task = Task.Run(async () =>
+            {
+                await Task.Delay(AskChatAboutContentPollTime * 1000 + 5000);
+
+                do
+                {
+                    var result = await ChatClient.CheckPollAsync(AskChatAboutContentPollId);
+                    if (result.IsComplete && AskChatAboutContentCheckPollResults)
+                    {
+                        AskChatAboutContentCheckPollResults = false;
+
+                        if (result.IsSuccessful)
+                        {
+                            Tracker.Say(x => x.Chat.PollComplete);
+
+                            if ("Yes" == result.WinningChoice)
```

Comment:
I'd feel better about this if you were to use case insensitive comparison here (and in other places)